### PR TITLE
Updated for new table

### PR DESCRIPTION
**Pull request by fernandalavalle**
Fixed SQL query generation to refer to the new table: m_lab.ndt.all. Stripped out the logic for generating month specific tables and updated the tests to reflect this.

### CODE REVIEW

File: telescope/query.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def _create_select_clauses(metric):
 class BigQueryQueryGenerator(object):
 
     database_name = 'plx.google'
-    table_format = '[{database_name}:m_lab.{table_date}.all]'
+    table_format = '{database_name}:m_lab.ndt.all'
```
Comment (mtlynch) [inline telescope/query.py]: Oh neat! I don't know why we always did surrounding square brackets when it turns out they're not necessary. I guess nobody ever tried without them.

File: telescope/query.py [hunk 2]
```diff
@@ -135,7 +135,7 @@ def _create_select_clauses(metric):
 class BigQueryQueryGenerator(object):
 
     database_name = 'plx.google'
-    table_format = '[{database_name}:m_lab.{table_date}.all]'
```
Comment (mtlynch) [inline telescope/query.py]: (kind of out of scope, but as long as we're touching it) The `database_name` is always `plx.google` which means that the table name is constant and does not need to be constructed out of a format string. I suggest changing the call to `built_query_format.format` to just be:

`table='plx.google:m_lab.ndt.all'`

and deleting other table variables (`table_format`, `self._table`).

File: telescope/query.py (hunk 1 above)
Comment (fernandalavalle) [inline telescope/query.py]: yep. looked weird to me so I just got rid of it.

File: telescope/query.py (hunk 2 above)
Comment (fernandalavalle) [inline telescope/query.py]: fixed.

**Comment by mtlynch** [comment]
LGTM

**Pull request merged**